runtime/src/lib.rs

```diff
@@ -153,8 +160,24 @@ impl<'a> Drivers<'a> {
             crypto: DpeCrypto::new(&mut sha384),
             platform: DpePlatform,
         };
-        let dpe = DpeInstance::new(&mut env, DPE_SUPPORT)
+        let mut dpe = DpeInstance::new(&mut env, DPE_SUPPORT)
```

Comment:
I think it might be worth moving this into a helper function which creates the DPE instance, initializes it, and then returns it.